aten/src/ATen/native/AdaptiveAveragePooling.cpp

```diff
@@ -312,7 +312,12 @@ namespace {
     at::Tensor const& input,
     IntArrayRef output_size)
   {
-    auto output = at::empty({0}, input.options());
+    Tensor output;
+    if (input.is_quantized()) {
+      output = at::_empty_affine_quantized({0}, input.options());
```

Comment:
don't  you need to set quantizer here?

Reply:
I believe empty_affine_quantized takes scale and zero_point as well